.github/workflows/build.yml

```
@@ -0,0 +1,135 @@
+        id: create_release
+        uses: ncipollo/release-action@v1
+        with:
+          name: ${{ env.VERSION }}
+          artifacts: ./DEV-VERSION
+          token: ${{ secrets.GITHUB_TOKEN }}
+          tag: ${{ env.VERSION }}
+          allowUpdates: true
+          prerelease: true
+          body: ${{ env.VERSION }}
+        env:
+          export PATH: $PATH:/usr/local/go/bin
```

actionlint (1.7.4)

environment variable name "export PATH" is invalid. '&', '=' and spaces should not be contained

(env-var)